ajax/libs/jquery.scrollspeed/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.scrollspeed",
+  "filename": "jQuery.scrollSpeed.min.js",
+  "description": "Extension for custom scrolling speed and easing",
+  "version": "1.0.2",
+  "author": "Nathan Rutzky",
+  "homepage": "https://code.nath.co/scrollSpeed",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brightcove/videojs-thumbnails.git"
+  },
+  "keywords": [
+    "jquery",
+    "scrolling",
+    "easing"
+  ],
+  "license": "MIT"
```

Comment:
How did you know this lib is under MIT license?
Is there any reference?
Thanks